README.md

```diff
@@ -33,13 +33,31 @@ yarn
 Define a `.env` file and define, at minimum, the following (see [here](./src/lib/env.ts) for more settings):
 
 ```
-OVAL_ADDRESS=0x420 // The address of the Oval contract you want to target with the Oval Node.
-SENDER_PRIVATE_KEY=0x6969 // The private key of the permissioned actor who can call unlockLatestValue on the associated Oval.
-AUTH_PRIVATE_KEY=0x6666 // The root private key used to derive searcher specific keys for signing bundles.
-REFUND_ADDRESS=0x42069 // The refund address you want to send the OEV kickback to.
-PROVIDER_URL=https://mainnet.infura.io/v3/abcdef // Ethereum mainnet RPC provider.
+SENDER_PRIVATE_KEY=<your_sender_private_key>  # Private key of the actor authorized to call unlockLatestValue on the Oval.
+AUTH_PRIVATE_KEY=<your_auth_private_key>      # Root private key for deriving searcher-specific keys for signing bundles.
+PROVIDER_URL=<your_provider_url>              # Ethereum mainnet/goerli RPC provider URL.
+BUILDERS=<builders_json_array>                # JSON array specifying the builders for MEV-Share.
+CHAIN_ID=<network_chain_id>                   # Chain ID of the Ethereum network you are targeting, 1 or 5 for mainnet or goerli.
+OVAL_CONFIGS=<oval_configs_json>              # JSON string that maps Oval contract addresses to their specific configurations.
 ```
 
+`OVAL_CONFIGS` is a JSON string that maps Oval contract addresses to their specific configurations. Each entry in this JSON object should have the following format:
```

Comment:
is this an array? if so its not clear in the example

Reply:
No, Its JSON object where keys are Oval contract addresses mapping to nested config object